From b0a179fa237e696f99c488ffebacb219d9b6a257 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Tue, 11 Sep 2018 14:58:34 +0800
Subject: [PATCH 4582/5242] MLK-19537-1 arm64: fsl-imx8mm.dtsi: use gpr node
 for MIPI CSI PHY reset

commit  02dfb90df87bab15d74777f40b76d14807bd757f from
https://source.codeaurora.org/external/imx/linux-imx.git

Use GPR node to simplify codes for MIPI CSI phy reset

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Reviewed-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 0f576c50e71cadf43755d44d102beb813595d007)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 79a7c2a..601c2de 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -306,6 +306,7 @@
 			<&clk IMX8MM_CLK_DISP_APB_ROOT>;
 		clock-names = "mipi_clk", "phy_clk", "disp_axi", "disp_apb";
 		bus-width = <4>;
+		csi-gpr = <&dispmix_gpr>;
 		power-domains = <&mipi_pd>;
 		status = "disabled";
 	};
-- 
1.7.9.5

